build_rootfs_riscv64.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+# This script is expected to be synced with `build_container.sh` in the feature,
```

Comment:
We might want to do this right from the get-go, because there is quite a bit of duplication. Maybe we can factor installation of common dependencies and the toolchain definitions into a shared script, to make sure they don't go out of sync between risc-v and the other archs?

although I guess that might be a bit difficult, with this being a rootfs, and build_container building a container userland :thinking:

Reply:
I've tried to use `build_container.sh` in `riscv64` containers, it failed in my second draft unfortunately (it's some kind of permission problem while installing `lib-pipewire`), I will try it here and work it out hopefully 🧐

Reply:
Yes, I don't expect the entirety of `build_container` to work on riscv64 (and don't think it needs to for the initial PR). I was more thinking to have the parts that _do_ work inside of a separate scripts thats then called from both `build_rootfs` and `build_container` :)

Reply:
Ah, just saw what you pushed, this works too! Let's see what the CI says